src/oas3/transformers/request.ts

```diff
@@ -48,7 +48,7 @@ export const translateRequestBody = withContext<
     Optional<IHttpOperationRequestBody<true> | Reference>
   >
 >(function (key, requestBodyObject) {
-  const maybeRequestBodyObject = this.maybeResolveLocalRef(requestBodyObject) ?? requestBodyObject;
+  const maybeRequestBodyObject = this.maybeResolveLocalRef(requestBodyObject);
```

Comment:
why remove this fix? Is it not needed anymore?

Reply:
Yikes!  Reverted in e93cc80.